### PR TITLE
fix(ui): Inconsistent Header Height

### LINKED ISSUE #1653: Header inconsistent height for logged/not logged users
There's slight difference in the height of header, depending if user is logged in or not. It'd be good to have it the same for both states.

imageimage

### PR DESCRIPTION
**Pull request by jaitjacob**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #1653

### CODE REVIEW

**Review by Sboonny — CHANGES_REQUESTED** [review]
The issue still presist

imageimage

**Comment by Sboonny** [comment]
I am wondering is it better to change the avatar component width and height, instead of the Header in `/client/src/components/PageLayout/Header.tsx`?

**Comment by jaitjacob** [comment]
> 

@Sboonny I did change the Avatar(Avatar.tsx) width and height **NOT** Header(Header.tsx). Not sure where did I go wrong.

**Comment by Sboonny** [comment]
True you have, but the header's height still changes because `50px`, are bigger than the header height. You can use `36px` in the avatar, but it won't be presentable.

**Comment by jaitjacob** [comment]
@Sboonny thanks for the clarification. I think this should pass.

File: client/src/components/PageLayout/Header.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ const HeaderItem = forwardRef<HTMLDivElement, Props>((props, ref) => {
       {...props}
       w="full"
       as="header"
+      height={'90px'}
```
Comment (Sboonny) [inline client/src/components/PageLayout/Header.tsx:33]: The increased spaces seems off-putting to me in small screens. what about `4.5em`, so it looks like this 

image


```suggestion
      height={'4.5em'}
```

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @jaitjacob, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Comment by Sboonny** [comment]
@allcontributors add @jaitjacob for code

**Pull request merged**